_scss/_utilities.scss

```diff
@@ -58,6 +58,9 @@ i.fa.fa-outdent {
     padding: 0px 15px 10px 15px;
 }
 
+.image-inline img {
+    display: inline;
```

Comment:
@mstanleyjones I tried `inline`, `inline-block`, and `auto` (which Josh started with). They all work if they specified directly in the file as `style` but nothing works if I try to call the class in `_scss/_utilities.scss`